ReactAndroid/src/main/java/com/facebook/react/views/drawer/ReactDrawerLayoutManager.java

```diff
@@ -90,6 +93,21 @@ public void setDrawerLockMode(ReactDrawerLayout view, @Nullable String drawerLoc
   }
 
   @Override
+  public void setElevation(ReactDrawerLayout view, float elevation) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+      // Facebook is using an older version of the support lib internally that doesn't support
+      // setDrawerElevation so we invoke it using reflection.
+      // TODO: Call the method directly when this is no longer needed.
+      try {
+        Method method = ReactDrawerLayout.class.getMethod("setDrawerElevation", float.class);
+        method.invoke(view, PixelUtil.toPixelFromDIP(elevation));
+      } catch (Exception e) {
+        // Ignore errors when the method is not present in older versions of support lib.
```

Comment:
I wasn't sure how to handle the error here but I don't think it can happen for users outside facebook since the gradle build already has a dependency on a recent version of the support lib that always has the method so I'm not sure if logging something is needed.

`getMethod` cannot return null, it will throw if it cannot find the method or something else is wrong.